Java/sort/ShellSort.java

```diff
@@ -0,0 +1,52 @@
+class ShellSort 
+{ 
+	static void printArray(int arr[]) 
+    { //function to print the elements of the array
+        //called twice...one before sorting and once after sorting
```

Comment:
Doesn't look grammatically correct to me. Please keep in mind, that the comments are not only for your own reference but also for the reference of others.